src/node/Main.hs

```diff
@@ -73,7 +73,10 @@ action args@SimpleNodeArgs {..} = do
         sscParams = bool (Left ()) (Right gtParams) (CLI.sscAlgo commonArgs == GodTossingAlgo)
     case (sscParams) of
         (Left par)  -> actionWithoutWallet @SscNistBeacon par currentParams
-        (Right par) -> actionWithoutWallet @SscGodTossing par currentParams
+        (Right par) ->
+            -- TODO: What the heck?!
+            -- @actionWithoutWallet@ requires HasCoreConstants, why?
+            giveStaticConsts (actionWithoutWallet @SscGodTossing par currentParams)
```

Comment:
That's what I'm saying. The constraint are `(SscConstraint ssc, SecurityWorkersClass ssc)` and `ssc ~ SscGodTossing`, so we can conclude that the compiler thinks that `HasCoreConstants` is somehow necessary to get `(SscConstraint SscGodTossing, SecurityWorkersClass SscGodTossing)`.